recipes/typer-cli/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "typer-cli" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace('_', '-') }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 55b9794f347311e9c2fb1ce4b10a66945cd95f2f30e2452b1bf96c3ab3163632
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  entry_points:
+    - typer = typer_cli.main:main
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python
+  run:
+    - python ~=3.6
+    - importlib_metadata ~=1.5
```

Different style (but preferred), same meaning
```suggestion
    - importlib_metadata 1.5
```

---

I transform to `importlib_metadata >=1.5,<2` to be more explicit